x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_expression.tsx

```diff
@@ -28,12 +28,12 @@ export interface MetricRender {
   value: MetricChartProps;
 }
 
-export const metricChart: ExpressionFunction<
+export const metricChart: ExpressionFunctionDefinition<
   'lens_metric_chart',
   LensMultiTable,
-  MetricConfig,
+  Omit<MetricConfig, 'layerId'>,
```

Comment:
Otherwise there is this error, because `layerId` is missing in `args`:

![image](https://user-images.githubusercontent.com/9773803/73765670-283c2f00-4775-11ea-88f9-c335fc3d399d.png)